client/routes/Onboarding.tsx

```diff
@@ -26,6 +30,7 @@ const useOnboarded = () => {
     const {onboarded} = await response.json();
     if (onboarded) {
       refetch();
+      refetchAccountSession();
```

Comment:
Right now the `Finance` tab only shows up if Platforms owns loss liability and Stripe collects fees from Platform. 

After landing on the tab, account clicks `Enable` and they're re-onboarded to accept some TOSes. Afterwards, we need to refresh/refetch the account session now that the account can use the finance related components.